app/components/Onboarding/Steps/ConnectionTypeItem.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { opacity } from 'styled-system'
+import { Box as BaseBox, Flex as BaseFlex } from 'rebass'
+import { Card as BaseCard, Radio, Text, Heading } from 'components/UI'
+
+import ConnectOnboarding from 'components/Icon/ConnectOnboarding'
+import ImportOnboarding from 'components/Icon/ImportOnboarding'
+import PlusOnboarding from 'components/Icon/PlusOnboarding'
+import BoltOnboarding from 'components/Icon/BoltOnboarding'
+
+import { animated, Transition } from 'react-spring'
+
+const Box = styled(BaseBox)(opacity)
```

Comment:
Outside of the scope of this PR, but we might want to think about creating our own `Box` component that extends @rebass.Box and adds support for some of the other props that we commonly use, such as opacity and height.

Reply:
You are right, but actually that code wasn't even needed in the current layout design. So I removed it